src/http-auth-interceptor.js

```diff
@@ -122,7 +122,10 @@
        */
       retryAll: function(updater) {
         for (var i = 0; i < buffer.length; ++i) {
-          retryHttpRequest(updater(buffer[i].config), buffer[i].deferred);
+          var _cfg = updater(buffer[i].config);
+          if (_cfg === false)
```

Comment:
Why won't you call `rejectAll` instead? Is is that there are some events you want to abandon and some you want to be retried? What's the case?